packages/wallets/src/features/accounts/screens/VerifyDocumentDetails/VerifyDocumentDetails.scss

```diff
@@ -76,4 +90,17 @@
             padding: 0 0 1.6rem;
         }
     }
+
+    &__dummy {
```

Comment:
is there a better className for this rather than `__dummy` sounds abit too harsh 😂 
It's my personal opinion if there's a reason for it then you can just ignore this